src/Mammon/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Mammon.Extensions
+{
+    public static class StringExtensions
+    {
+        public static string ToResourceActorId(this string value) 
+        {
+            const string providers = "/providers";
+
+            //we want to extract only the top level provider
```

Comment:
would extract a method named "getTopLevelProvicer" avoid the need for the comment?